src/providers/oracle/qgsoracleprovider.cpp

```diff
@@ -2451,7 +2577,12 @@ QgsRectangle QgsOracleProvider::extent() const
       if ( mUseEstimatedMetadata )
       {
         // TODO: make SDO_DIMNAME values configurable (#16252)
-        if ( exec( qry, QStringLiteral( "SELECT sdo_lb,sdo_ub FROM mdsys.all_sdo_geom_metadata m, table(m.diminfo) WHERE owner=? AND table_name=? AND column_name=? AND sdo_dimname='X'" ),
+
+        const QString sql { QStringLiteral( "SELECT sdo_lb,sdo_ub FROM mdsys.all_sdo_geom_metadata m, table(m.diminfo) WHERE owner=? AND table_name=? AND column_name=? AND sdo_dimname='X'" ) };
+
+        QgsDatabaseQueryLogWrapper logWrapper { sql, mUri.uri(), QStringLiteral( "oracle" ), QStringLiteral( "QgsOracleProvider" ), QGS_QUERY_LOG_ORIGIN };
```

Comment:
Why not using the macro LoggedExecStatic everytime we need to execute a request?

Reply:
The main reasons for using the wrappers instead of the macro are:

1. when a statement is prepared
2. when we need more control over the logger, for example when we have customized error messages or when we need to report the number of rows.

I think I have used the wrapper only when it was required but I'll have a look if I overlooked something.

Reply:
I've removed most wrappers.